### DISCUSSION TITLE
Pause Scrolling App Labels in Quick Panel

### DISCUSSION THREAD

**Discussion by pojesh** [salvogiangri/UN1CA#151] [Feature requests]
<img loading="lazy" src="https://github.com/user-attachments/assets/6a5ef3dc-0b83-420e-800e-190a2c7e3470" width="25%"/><br>

Pause these labels in quick settings .. it's annoying 

+ Shadisnx once said it spikes the CPU usage while pulling the quick panel down which makes it stutter

**Comment by salvogiangri** [comment]
> * Shadisnx once said it spikes the CPU usage while pulling the quick panel down which makes it stutter

Yes, this was initally found by @ananjaser1211 by doing tests on 9810 and confirmed to be happening on my devices with 6.1 as well. We'll have to see if people are okay with me removing text marquee tho :/

**Reply by Reinsworth** [reply]
Maybe if possible add toggle for it like native blur, since text marquee is kind of usefull in some case. Im sure some user will prefer to keep it.

**Comment by salvogiangri** [comment]
Closing this as it has been implemented in the ROM and will be available from the next release.

**Reply by pojesh** [reply]
Gr8 thanks
